### DISCUSSION TITLE
openai开源马拉松群

### DISCUSSION THREAD

**Discussion by wetwowo** [Grt1228/chatgpt-java#128] [General]
群内有清华，川大的大佬，欢迎各位朋友入群
![微信图片_20230427212500](https://user-images.githubusercontent.com/131851517/235339475-4881b5da-1f91-4561-977f-710cef5b21e1.jpg)
二维码过期的话加我，我拉你入群
![微信图片_20230428144227](https://user-images.githubusercontent.com/131851517/235339483-9ca2b7db-a0e3-48b3-bd5c-222a96d46a79.jpg)

**Comment by zihao2981** [comment]
老板加你了，同意下
